### DISCUSSION TITLE
Error Using LaravelLocalization in prefix

### DISCUSSION THREAD

**Discussion by PterodactylStore** [livewire/livewire#6035] [7. Bugs (archived)]
![image](https://github.com/livewire/livewire/assets/133764150/f6d7bee9-02c1-424f-85c3-1aacff16997b)

im using this but when try to change lang i get the error in the screenshot
```php
Route::group(['prefix' => LaravelLocalization::setLocale()], function ()
{
    // Your other localized routes...
 
    Livewire::setUpdateRoute(function ($handle) {
        return Route::post('/livewire/update', $handle);
    });
});
```

**Comment by joshhanley** [comment]
@PterodactylStore thanks for reporting! Any chance you could provide an example repo that demonstrates the issue with as little code as possible? Thanks!
